src/cloudabi-run/yaml_file_descriptor_factory.h

```diff
@@ -34,6 +34,7 @@ class YAMLFileDescriptorFactory
 
  private:
   YAMLFactory<const argdata_t *> *const fallback_;
+  const int execfd_ = -1;
```

Comment:
Could you please remove the initialization? The constructor guarantees that this field is always initialized.